routes/web.php

```diff
@@ -36,6 +36,7 @@
 
     Route::get('/print', 'PrintController@index')->name('print');
     Route::post('/print/modify_balance', 'PrintController@modifyBalance')->name('print.modify')->middleware('can:print.modify');
+    Route::post('/print/send_pages', 'PrintController@sendPages')->name('print.send');
```

Comment:
There is confusion all over the place.
If we transfer balance (money), why we call this feature _sending pages_...

If we are sending money, we should be clear about that and not put it under the _print_ section.
Maybe a link with a description and a logo would do the job redirecting the user to _transferring balance_ page.